packages/flutter_tools/test/general.shard/build_system/targets/icon_tree_shaker_test.dart

```diff
@@ -549,7 +549,7 @@ const String validFontManifestJson = '''
 
 const String invalidFontManifestJson = '''
 {
-  "famly": "MaterialIcons",
```

Comment:
Was it intended wrong to test abnormal scenarios? I am too sure.

Reply:
You are probably right. I already encountered the same thing in on of my previous PRs. I will redo this just in case.